diff --git a/node_modules/@arcasdk/pdf/lib/generator/invoice-pdf-generator.js b/node_modules/@arcasdk/pdf/lib/generator/invoice-pdf-generator.js
index 2c7448f..47c3e2e 100644
--- a/node_modules/@arcasdk/pdf/lib/generator/invoice-pdf-generator.js
+++ b/node_modules/@arcasdk/pdf/lib/generator/invoice-pdf-generator.js
@@ -115,7 +115,7 @@ class InvoicePdfGenerator {
             const renderTemplate = this.renderTemplate;
             const optsWithCopy = Object.assign(Object.assign({}, this.opts), { copy });
             const html = renderTemplate({ data, options: optsWithCopy, qrDataUrl });
-            const browser = yield puppeteer_1.default.launch({ headless: true });
+            const browser = yield puppeteer_1.default.launch({ headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox'] });
             try {
                 const page = yield browser.newPage();
                 const { format: pageFormat, width: pageWidthMm, height: pageHeightMm, } = PAGE_DIMENSIONS[this.opts.pageSize || "A4"] || PAGE_DIMENSIONS.A4;
